packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -143,6 +143,11 @@ const cli = {
         describe: 'Whether or not to automatically update all reference screenshots with the latest screenshots.',
         default: false,
       },
+      useSeleniumStandalonService: {
+        type: 'boolean',
+        describe: 'A flag to use the selenium standalone service instead of the selenium docker service.',
+        default: () => process.env.USE_SELENIUM_STANDALONE_SERVICE === 'true',
```

Comment:
can we just check the existence of the env?

```suggestion
    default: process.env.USE_SELENIUM_STANDALONE_SERVICE
```

Reply:
envs are always a string and the type here needs to be a boolean. Otherwise, it would default to a string of 'true'.

Reply:
boooo

Reply:
do we still need to execute this as a function though?
```suggestion
        default: process.env.USE_SELENIUM_STANDALONE_SERVICE === 'true',
```

Reply:
You win!